pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountThetaSketchAggregationFunction.java

```diff
@@ -57,11 +61,31 @@
  * <p>TODO: For performance concern, use {@code List<Sketch>} as the intermediate result.
  */
 public class DistinctCountThetaSketchAggregationFunction implements AggregationFunction<Map<String, Sketch>, Long> {
+
+  public enum MergeFunction {
```

Comment:
I feel putting these functions as constant is more readable and easier to use:
`public static final String UNION = "UNION";`
You can directly `switch` on the `expression.getFunction().getFunctionName().toUpperCase()` and not need to worry about `MergeFunction.valueOf()` throws exception.

Reply:
I got rid of the unit testing method as you suggested. But I personally like to keep the enums for type-safety. They don't really have any overhead since they are all static right? Happy to change if you feel strongly.